src/Components/AuthRedirectWrapper.tsx

```diff
@@ -13,7 +13,9 @@ interface AuthRedirectWrapperProps {
  * @param children - JSX element that will be rendered if the user is unauthenticated.
  * @returns The authentication redirect component.
  */
+// sonarignore:start
```

Comment:
What is the need of this sonar ignore?

Reply:
Because the sonar issue was about nomenclature of the constant, where sonar was saying the first alphabet has to be small, but react didn't allow it